src/main/java/org/gridsuite/loadflow/server/annotations/PostCompletion.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server.annotations;
```

Comment:
You can move annotations package in the utils package. So it is homogeneous with other micro services

Reply:
Done